app/helpers/components/filter_helper.rb

```diff
@@ -4,6 +4,7 @@ def filter_icon(&block)
   end
 
   def render_filter(items, **options, &block)
+    options[:mode] = :starts_with unless [:includes, :starts_with].include?(options[:mode])
```

Comment:
this could be a boolean, but I couldn't come up with a name hehehe

Reply:
Maybe `regex_enable`?